lenskit-eval/src/main/java/org/lenskit/eval/traintest/ExperimentJob.java

```diff
@@ -22,6 +22,7 @@
 
```

Comment:
Uggh the diff on this class is miserable. Given only a cursory review this appears to mosty a restructure of already existing code plus added blocking on the semaphore.